README.md

```diff
@@ -47,19 +45,35 @@ Creates a new plugin at the given path.
 
 ### `build`
 
-Builds your current package based on the configuration in your `package.json` and `packup.config.ts` (if applicable).
+Builds your current package based on the configuration in your `package.json`.
```

Comment:
Do we not use the packup.config.ts? I thought we did.

Reply:
it's used internally but there is nothing really done for loading it and I would rather avoid for now until we decide what we do moving forward